docs/privacy-policy.md

```diff
@@ -0,0 +1,172 @@
+Privacy Policy
+==============
+
+**DRAFT**
+
+Your personal privacy is an important factor that the Sempervirens project and Numfocus 
+(our legal entity) considers in the development of each of our products and services. 
+We are committed to NOT storing ANY personal information through Sempervirens other 
+than IP addresses. We are committed to being transparent and open. This Privacy Policy 
+explains generically how we receive information from you, and what we do with that 
+information once we have it.
+
+What do we mean by "personal information?"
```

Comment:
We might want to use the phrase "personally identifying information" so it includes other non-personal information that could still be used to track back to the person in some way.